svc-api/main.go

```diff
@@ -66,20 +66,32 @@ func main() {
 				yes := strings.Contains(basicAuth, "Basic")
 				if yes {
 					spl := strings.Split(basicAuth, " ")
+					if len(spl) != 2 {
+						errorMessage := "Invalid basic auth provided"
+						log.Error(errorMessage)
+						invalidAuthResp(errorMessage, w)
+						return
+					}
 					data, err := base64.StdEncoding.DecodeString(spl[1])
 					if err != nil {
+						errorMessage := "Error during decoding the authorization : " + err.Error()
```

Comment:
please give an error message without the word `Error`.
log.Error(err.Error()) will add word `Error`, so it will be redundant.

Reply:
Changed the error message